crates/agglayer-storage/src/storage/mod.rs

```diff
@@ -29,7 +29,7 @@ pub enum DBError {
     #[error("Trying to access an unknown ColumnFamily")]
     ColumnFamilyNotFound,
 
-    #[error("Database is opened in read-only mode")]
+    #[error("Database was opened in read-only mode")]
```

Comment:
in datadog people might wonder how is this an error, maybe let's precise that we got a write attempt

```suggestion
    #[error("Cannot write in database opened in read-only mode")]
```